Developer-guide/Development-Workflow.md

```diff
@@ -187,26 +157,13 @@ generate the build configuration:
 
 #### GlusterFS
 
-Ubuntu:
-
-Type the following to build and install GlusterFS on the system:
-
-        sh$ make
-        sh$ make install
-
-CentOS/RHEL/Fedora:
-
-In an rpm based system, there are two methods to build GlusterFS. One is
-to use the method describe above for *Ubuntu*. The other is to build and
-install RPMS as described in [CompilingRPMS](./Compiling-RPMS.md).
+ sh$ make install
 
 #### GlusterFS UFO/SWIFT
 
 To build and run Gluster UFO you can do the following:
 
-1.  Build, create, and install the RPMS as described in
-    [CompilingRPMS](./Compiling-RPMS.md).
-2.  Configure UFO/SWIFT as described in [Howto Using UFO SWIFT - A quick
+1.  Configure UFO/SWIFT as described in [Howto Using UFO SWIFT - A quick
```

Comment:
The link can be updated to this one: https://github.com/gluster/gluster-swift/blob/master/doc/markdown/quick_start_guide.md

Reply:
Moved it to another place, and updated the Link